src/cli.mjs

```diff
@@ -42,15 +51,44 @@ export async function run(_argvInput, _container) {
     let subMessage = chalk.reset(
       "Use 'fauna <command> --help' for more information about a command.",
     );
+    let epilogue = "";
```

Comment:
thoughts on extracting this to a helper function and typing the error object? might make it easier to recall/find how to get certain error behaviors.

edit: found the types. still think this might be worth extracting just because our runner/entrypoint is now like 90% error handler.